en/components/grids_templates/remote-data-operations.md

```diff
@@ -789,7 +709,7 @@ The last step will be to declare our template for the gird.
 After all the changes above, the following result will be achieved.
 
 @@if (igxName === 'IgxGrid') {
-<div class="sample-container loading" style="height:620px">
+<div class="sample-container loading" style="height:400px">
```

Comment:
WIth 400px the sample is not fully visible:

![image](https://user-images.githubusercontent.com/11193764/108325008-4b474e80-71d1-11eb-9e63-f9bdf4daa500.png)